components/accelerator/nvidia/query/fabric-manager-log/log.go

```diff
@@ -0,0 +1,15 @@
+package fabricmanagerlog
+
+const (
+	// e.g.,
+	// [Jul 23 2024 07:53:55] [ERROR] [tid 841] detected NVSwitch fatal error 20034 on fid 0 on NVSwitch pci bus id 00000000:86:00.0 physical id 3 port 33
+	RegexNVSwitchFatalSXidFromLog = `.+detected NVSwitch fatal error (\d+)`
```

Comment:
When will the Regex got compiled? Since these are fixed string, could we use regexp.MustCompile to compile the regexp once instead of compiling multiple times at runtime?

Reply:
We pass these to the poller config and compile once here:

https://github.com/leptonai/gpud/blob/c60a71308be289704d110a42fd10d247a39abe63/components/query/log/filter/filter.go#L54-L73